av1an-core/src/concat.rs

```diff
@@ -264,6 +220,19 @@ pub fn mkvmerge(
   Ok(())
 }
 
+/** This code adds the necessary command parameters to the options FILE
+ * that is passed in, up to the chunk NUM. Returns nothing.
+ */
+pub fn concatenate_mkvmerge(num: u64, mut this_file: File) {
+  let mut file_string: String = "[ \"-o\", \"output.mp4\", \"audio.mp3\", \"[\"".into();
```

Comment:
We want to make sure we're using the same paths as before, e.g. where we were using `fix_path(output)`.